AZ3166/src/libraries/AzureIoT/examples/RemoteMonitoringv2/RemoteMonitoring.ino

```diff
@@ -0,0 +1,226 @@
+
+bool sendDeviceInfo()
+{
+  char reportedProperties[2048];
+  snprintf(reportedProperties,2048, twinProperties,roomSchema, interval ,roomSchema,deviceType,deviceFirmware,deviceFirmwareUpdateStatus,deviceLocation,deviceLatitude,deviceLongitude);
+  return DevKitMQTTClient_ReportState(reportedProperties);
+}
+
+void twinCallback(DEVICE_TWIN_UPDATE_STATE updateState, const unsigned char *payLoad, int length){
+  LogInfo("*** Twin State: %s",updateState?"Complete":"Partial");
+}
+
+RGB_LED rgbLed;
+uint8_t colors[][3] = {
+        {255, 0, 0},                // red
+        {0, 255, 0},                // green
+        {0, 0, 255},                // blue
+        {0, 0, 0},
+        {255, 255, 0},
+        {0, 255, 255},
+        {255, 0, 255},
+        {255, 255, 255}
+    };
+uint8_t colorindex;
+bool doReset=false;
+
+int device_method_callback(const char *methodName, const unsigned char *payload, int length, unsigned char **response, int *responseLength){
+
+  LogInfo("*** Remote method: %s",methodName);  
+
+  if(strcmp(methodName,"LedColor")==0){
+    int n=sizeof(colors)/sizeof(colors[0]);
+    uint8_t *color=colors[(colorindex++)%n];
+    rgbLed.setColor(color[0],color[1],color[2]);
+
+    const char *ok="{\"result\":\"OK\"}";
+    *responseLength=strlen(ok);
+    *response = (unsigned char*)malloc(*responseLength);
+    strncpy((char *)(*response), ok, *responseLength);
+    return 200;
+  }
+
+  if(strcmp(methodName,"Reboot")==0){
+    doReset=true;
+    
+    const char *reset="{\"result\":\"RESET\"}";    
+    *responseLength=strlen(reset);
+    *response = (unsigned char*)malloc(*responseLength);
+    strncpy((char *)(*response), reset, *responseLength);
+    return 201;
+  }
+
+  LogError("*** Remote method: %s not found",methodName);
+  return 500;
+}
+
+void InitWiFi()
+{
+  Screen.print("WiFi \r\n \r\nConnecting...\r\n             \r\n");
+  
+  if(WiFi.begin() == WL_CONNECTED)
+  {
+    IPAddress ip = WiFi.localIP();
+    sprintf(wifiBuff, "WiFi \r\n %s\r\n %s \r\n \r\n",WiFi.SSID(),ip.get_address());
+    Screen.print(wifiBuff);
+    digitalWrite(LED_WIFI, 1);
+
+    DevKitMQTTClient_SetDeviceMethodCallback(&device_method_callback);
+    DevKitMQTTClient_SetDeviceTwinCallback(&twinCallback);
```

Comment:
Initializing MQTT client and sending message should not be part of InitWifi(). Could you move them into Setup() directly?